src/by/epam/lab/PurchaseUtils.java

```diff
@@ -0,0 +1,45 @@
+package by.epam.lab;
+
+public class PurchaseUtils {
+    private final Purchase purchase;
+
+    public PurchaseUtils() {
+        purchase = new Purchase();
+    }
+
+    public PurchaseUtils(Purchase purchase) {
+        this.purchase = purchase;
+    }
+
+    public Purchase getPurchase() {
+        return purchase;
+    }
+
+    public void printPurchase() {
+        System.out.printf("%s%n", purchase);
+    }
+
+    public void printCost() {
+        System.out.printf("cost = %s%n", purchase.getCost());
+    }
+
+    public void printCostDiff(Purchase purchase) {
+        Byn costDiff = new Byn(getPurchase().getCost().compareTo(purchase.getCost()));
+        System.out.printf("diff = %s BYN%n", costDiff);
+    }
+
+    public void printIsSameCost(Purchase[] purchases) {
```

Comment:
add number of arguments in the signature, not an array.
printIsSameCost(Purchase ... purchases)